package.json

```diff
@@ -4,14 +4,14 @@
   "main": "index.js",
   "license": "MIT",
   "devDependencies": {
-    "prisma": "2.24.1",
-    "netlify-cli": "3.37.1"
+    "netlify-cli": "3.37.1",
+    "prisma": "^3.14.0"
   },
   "scripts": {
     "build": "prisma generate",
     "dev": "netlify dev"
   },
   "dependencies": {
-    "@prisma/client": "2.24.1"
+    "@prisma/client": "^3.14.0"
```

Comment:
Looks like we could update Prisma to `4.7.1` and netlify-cli to `[12.2.9](https://www.npmjs.com/package/netlify-cli/v/12.2.9)`